### PR TITLE
fix(styles): resolve style issues

### PR DESCRIPTION
**Pull request by josephmcg**
**What this PR does** 📖
- increase size of sidebar toggle component. add color
- move sidebar toggle for friends page outside specific route upto tab level
- remove unused global styles
- improve styles of select component

**Which issue(s) this PR fixes** 🔨

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by WanderingHogan** [comment]
@jasonwoodland @nathan-power the markup looks good to me, but there is a lot of CSS changes; does it all look okay to you guys?

**Comment by nathan-power** [comment]
Hey @WanderingHogan! I've taken a quick look and it looks good to me, but it also touches on a lot of the button stuff, which @jasonwoodland has updated recently. I'll let him take a look as well to make sure that there aren't any issues 👍

**Comment by jasonwoodland** [comment]
Found a few things regarding button/selelect controls, making the changes now.

File: components/interactables/Button/Button.vue [hunk 1]
```diff
@@ -47,7 +48,7 @@ export default Vue.extend({
         backgroundPosition: `left ${this.cursorX - 150}px top ${
           this.cursorY - 150
         }px`,
-        opacity: this.showCursorGlow ? 1 : 0,
+        opacity: !this.disabled && this.showCursorGlow ? 1 : 0,
```
Comment (jasonwoodland) [inline components/interactables/Button/Button.vue:51]: Add this so the cursor glow wouldn't be stuck once the disabled prop was added

**Review by jasonwoodland — APPROVED** [review]
Fixed a few extra outstanding style issues, LGTM!

**Review by nathan-power — APPROVED** [review]
@jasonwoodland Checked your changes and they seem good! Especially the transition for the dropdowns.

**Pull request merged**